pkg/ui/ui.go

```diff
@@ -158,6 +159,11 @@ func Handler(cfg Config) http.Handler {
 	buildInfo := build.GetInfo()
 
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		licenseType, err := base.LicenseType(cfg.Settings)
+		if err != nil {
+			log.Errorf(context.Background(), "unable to get license type: %+v", err)
+		}
+		licenseTTL := base.GetLicenseTTL(r.Context(), cfg.Settings, timeutil.DefaultTimeSource{})
```

Comment:
This snippet can be omitted since this old version doesn't surface license alerts in the UI. I might add them back in when we make further changes.

Reply:
Any by "can" I mean "has to be" because it's a compiler error :)